editor/themes/editor_theme_manager.cpp

```diff
@@ -250,7 +250,7 @@ EditorThemeManager::ThemeConfiguration EditorThemeManager::_create_theme_config(
 	config.color_picker_button_height = 28 * EDSCALE;
 	config.subresource_hue_tint = EDITOR_GET("docks/property_editor/subresource_hue_tint");
 
-	config.default_contrast = 0.3; // Make sure to keep this in sync with the editor settings definition.
+	config.default_contrast = 0.24; // Make sure to keep this in sync with the editor settings definition.
```

Comment:
Multiple themes use the default contrast, please adjust them accordingly so they do not have unintended visual changes.